dev/inventory/defaults.yaml

```diff
@@ -125,3 +126,20 @@ node_defaults:
     uri: /vms/images
   # ssh keys are also read and merged from ./ssh-keys
   ssh_keys: [ ]
+
+hardware_defaults:
```

Comment:
"For the next guy" add a comment with how to get this information, so Mr. Alice New-Programmer can find it easily on their machine.

```suggestion
# linux: for file in $(ls -1d /sys/bus/pci/drivers/*/0000\:*/net/*); do printf "% -6s %s\n" "$(basename $file)" $(grep PCI_ID "$(dirname $(dirname $file))/uevent" | cut -f 2 -d '='); done
# macOS: system_profiler -json SPUSBDataType
hardware_defaults:
```

I just copied some nonsense into the suggestion, it needs tuning.

Reply:
There's another option, but it needs lshw:

```
[root@lemon ~]# lshw -businfo -class network -disable scsi -disable usb -numeric
Bus info          Device          Class       Description
=========================================================
pci@0000:2a:00.0  enp42s0         network     RTL8125 2.5GbE Controller [10EC:8125]
```

Reply:
I've added comments to help Alice find the information.